include/quda_milc_interface.h

```diff
@@ -962,16 +962,9 @@ extern "C" {
    * @param[in] stopWtheta, 0 for MILC criterion and 1 to use the theta value
    * @param[in,out] milc_sitelink, MILC gauge field to be fixed
    */
-  void qudaGaugeFixingOVR( const int precision,
-    const unsigned int gauge_dir,
-    const int Nsteps,
-    const int verbose_interval,
-    const double relax_boost,
-    const double tolerance,
-    const unsigned int reunit_interval,
-    const unsigned int stopWtheta,
-    void* milc_sitelink
-    );
+  void qudaGaugeFixingOVR(const int precision, const unsigned int gauge_dir, const int Nsteps,
```

Comment:
I'm a bit confused here.  The function definition in milc_interface.cpp is lacking the `const` qualification on all these parameters, and yet the compiler isn't complaining.  Perhaps this is some "Cism" that I'm not aware of?

Regardless, I think the parameters should not be `const` qualified since it doesn't mean that much to `const` qualify a scalar that is passed by value.

Reply:
`const` and `volatile` parameters are ignored in function declarations, but they have an effect in function definition.

Reply:
One learns something new every day 🙂

Reply:
https://github.com/lattice/quda/commit/0138cf3566353710dd0e569f4060a76065a16290